packages/jest-enzyme/src/toMatchSnapshot.js

```diff
@@ -0,0 +1,21 @@
+import wrapperToJSON from 'enzyme-to-json';
+
+const GLOBAL_STATE = Symbol.for('$$jest-matchers-object');
+const originalToMatchSnapshot = global[GLOBAL_STATE].matchers.toMatchSnapshot;
```

Comment:
yeah. you shouldn't do this. Use get/set instead https://github.com/facebook/jest/blob/master/packages/jest-matchers/src/index.js#L183-L187